### ISSUE TITLE
Stockfish quite old

### ISSUE THREAD

**Issue by blablakali** [ml-research/liground#264]
Please add stockfish 16+ NNUE, for better analysis.
Thank you!

**Comment by QueensGambit** [comment]
I agree with your request. For now you can add a newer Stockfish executable manually.

**Comment by KudahShambare** [comment]
Which other engines would you guys want to be added? I have managed to add Stockfish 16. Currently running it on my local machine but will be publishing a pull request soon.

**Comment by henrileonard** [comment]
Newest release has Stockfish 17.
